### DISCUSSION TITLE
Updating macOS .nib translations

### DISCUSSION THREAD

**Discussion by bitigchi** [transmission/transmission#2418] [Q&A]
Hello,

As the author of Turkish macOS translations, I'd like to update the .nib files for the next release, however I am not sure if there are new files to be added before the release. When is the next release planned?

AFAICS, there is no motion in regards to move these translations to Transifex, so a roadmap would be nice to have.

**Comment by ckerr** [comment]
@livings124 @mikedld do either of you know the answer to this question?
